src/main/java/features/Cart.java

```diff
@@ -1,4 +1,88 @@
 package features;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Cart {
+
+    private ArrayList<Listing> listings;
+
+    /** A Constructor for Cart that sets contents of the cart to be itemListings.
+     *
+     * @param itemListings an ArrayList of unique Listings.
+     */
+    public Cart(ArrayList<Listing> itemListings){
+        this.listings = itemListings;
+    }
+
+    //OVERLOADED
+    /** A Constructor for Cart that takes no arguments*/
+    public Cart(){
+        this.listings = new ArrayList<Listing>();
+    }
+
+
+    /** Get all Listings in the cart.
+     *
+     * @return A shallow copy of the list of Listings within Cart (An ArrayList of Listings).
+     */
+    public ArrayList<Listing> getItems(){
```

Comment:
going forward, i don't think it's necessary to create a shallow copy and you probs don't have to be so thorough. that said, this is really smart and much safer. good job :)

Reply:
Thanks for the feedback man, I appreciate it immensely. I've fixed any styling inconsistency that I saw, just now. If there are any others, I don't see them and would appreciate you pointing them out.